octoprint_mrbeam/util/device_info.py

```diff
@@ -137,6 +138,42 @@ def get_beamos_version_number(self):
             self._logger.debug("beamos_version invalid: " + beamos_version)
             return None
 
+    def get_software_versions(self):
+        result = dict()
+        software_info = None
+
+        try:
+            plugin_info = self._plugin._plugin_manager.get_plugin_info("softwareupdate")
```

Comment:
this is softwareupdate_plugin_info right?

This is also the only place where we need the plugin... I don't think we need to store a reference to the plugin from the DeviceInfo. It seems like to initialise this class we need to pass any plugin to it... Doesn't feel like it belongs here... If we need the softwareupdate info  I think we should directly pass it in the constructor.
Another flag that shows that this is smelly is that we are directly calling the _plugin_manager which is a private variable which means it shouldn't be used outside of its own class.